src/hotspot/share/prims/jvmtiEnvThreadState.cpp

```diff
@@ -250,6 +257,14 @@ void JvmtiEnvThreadState::clear_frame_pop(int frame_number) {
   JvmtiEventController::clear_frame_pop(this, fpop);
 }
 
+void JvmtiEnvThreadState::clear_all_frame_pops() {
+#ifdef ASSERT
+  Thread *current = Thread::current();
+#endif
```

Comment:
I think you can just get rid of these lines and inline the Thread.current() call below.

Reply:
Okay, fixed.